config/locales/en/errors.yml

```diff
@@ -374,6 +374,10 @@ en:
           attributes:
             client_has_dependants:
               inclusion: Select yes if your client has any dependants
+        steps/income/dependants_form:
+          attributes:
+            dependants:
+              too_long: Must have fewer than 50 dependants
```

Comment:
Does this need to be 51? Not sure, just checking! - i.e. they can actually add 50, but not 51 right?